iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -85,6 +115,12 @@ TEST_F(ServiceDiscovery_test, FindServiceReturnsOfferedService)
     ASSERT_FALSE(serviceContainer.has_error());
     ASSERT_THAT(serviceContainer.value().size(), Eq(1U));
     EXPECT_THAT(*serviceContainer.value().begin(), Eq(SERVICE_DESCRIPTION));
+
+    sut.findService(SERVICE_DESCRIPTION.getServiceIDString(),
+                    SERVICE_DESCRIPTION.getInstanceIDString(),
+                    SERVICE_DESCRIPTION.getEventIDString(),
+                    lambda);
```

Comment:
Maybe a better name would be 

```suggestion
                    findHandler);
```

or

```suggestion
                    matchHandler);
```
as it is applied to all matching entries.
Applies to all tests.